crates/js-sys/tests/wasm/TypedArray.rs

```diff
@@ -16,6 +17,32 @@ macro_rules! each {
     )
 }
 
+macro_rules! test_inheritence {
+    ($arr:ident) => ({
+        let arr = $arr::new(&JsValue::undefined());
+        assert!(arr.is_instance_of::<$arr>());
+        assert!(arr.is_instance_of::<Object>());        
+    })
+}
+#[wasm_bindgen_test]
+fn inheritence() {
+    each!(test_inheritence);
+}
+
+macro_rules! each {
```

Comment:
This `each!` macro is the same as the one above it -- we should only use the original `each!` macro and delete this one.